### ISSUE TITLE
Issue with _ble_contrib_fzf_base introduced on 5aff5f6

### ISSUE THREAD

**Issue by 3ximus** [akinomyoga/blesh-contrib#26]
Hey there @akinomyoga ,
I've noticed that this commit introduces an issue for me. It defines an empty `_ble_contrib_fzf_base` which causes it to break my config. I have this on my `blerc` and it seems that this line 
https://github.com/akinomyoga/blesh-contrib/commit/5aff5f6cf5e18c9332a350bb6dee65e0ccf5d252#diff-d84137e6d92190596b730ec812c4bb7d93c4a0f15a04025df026200699e24f5aR12 ignores my custom base...

```bash
# blerc
_ble_contrib_fzf_base=~/.bash/fzf/
ble-import -d integration/fzf-completion
ble-import -d integration/fzf-key-bindings
```

I noticed that removing that line does the trick (since I'm guessing doesn't clear the variable loaded from blerc?)

Cheers

**Comment by akinomyoga** [comment]
Thank you for the report! I'll push a fix later.

**Comment by akinomyoga** [comment]
Sorry for the delay. I pushed a fix (1eee6ac4e3bf81f9ec9c7a67cc6231816ccd876c) to the `master` branch (which was actually a part of another change). Could you update ble.sh by `ble-update` and check the behavior? (Before an attempt of `ble-udpate`, you'd probably need to revert your local changes to `integration/fzf-initialize.bash` if any).

**Comment by 3ximus** [comment]
Yup, I can confirm that it works!! Thank you!
